docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+    cinema:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            - ./:/app
+            - my_media://files/media
+        command: >
+            sh -c "python manage.py wait_for_db &&
+                   python manage.py makemigrations &&
```

Comment:
There is no need to run `makemigrations` command, they should be already included